estimate.go

```diff
@@ -173,8 +175,44 @@ func estimate(importpath string) error {
 	return nil
 }
 
+func recursiveEstimate(importpath string, formatType string) error {
+	log.Print(aurora.Bold(aurora.Green("[DebGoGraph Starting]")))
+
+	debFilter := true
+	displayAll := true
+
+	fmt.Printf("\nGolang package: %s\n", aurora.Bold(importpath))
+	fmt.Printf("Output type: %s\n", aurora.Bold(formatType))
+	fmt.Printf("Deb filter: %v\n", aurora.Bold(debFilter))
+	fmt.Printf("Display all deb (false for only main package): %v\n\n", aurora.Bold(displayAll))
+
+	fmt.Printf("Fetching dependencies of %s | It may take a while.\n\n", aurora.Bold(aurora.BrightBlue(importpath)))
+
+	// List | Graph | Tree
+	m, err := gocheckdeb.GetDep(importpath, "imports", formatType)
```

Comment:
I'm not a fan of using many third-party libraries. Especially this library has some overlap functions with dh-make-golang.